Melt/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd `dirname $0`
+exec python3 chain.py -examine "$1"
```

Comment:
Note that I'm not using melt.py here because it gets in the way by running another copy of python (I needed it run run `python3`).

TBH, I'd be fine with melt.py going away (we don't need it) and making this the thing that you change if you want to call chain.py with different args.